Makefile

```diff
@@ -4,6 +4,10 @@ GO_TEST_FLAGS = $(VERBOSE)
 DEBUG = --debug
 SETUP = --no-setup
 
+# Local Operator SDK version and required version
+LOCAL_VERSION = $(shell operator-sdk version | head -n1 | cut -d" " -f3 | tr -d '",')
+REQUIRED_VERSION = v0.12.0
```

Comment:
this would need to be more specific naming, to show that it is about operator-sdk, not BMO

Reply:
wouldn't this fail if operator-sdk is not installed ?